data/strings/strings.txt

```diff
@@ -23342,54 +23248,54 @@
     zh-Hant = 載入地圖
 
   [pref_tts_speedcams_auto]
-    comment = Speed camera settings menu option - Warn (about speedcams) if risk of exceeding speed limit
-    tags = android
-    en = Warn if there is a risk of exceeding the speed limit
-    af = Waarsku indien daar ’n risiko is dat die grens oorskry word
-    ar = حذرني  إذا هناك احتمالية تجاوز الحد الأقصى للسرعة
-    az = Sürət həddini aşmaq riski varsa, xəbərdarlıq edin
-    be = Папярэджваць калі ёсць рызыка перавышэння дазволенай хуткасці
-    bg = Предупреждава ако има риск от превишаване на ограничението на скоростта
-    ca = Avisa si hi ha risc d'excedir el límit de velocitat
-    cs = Upozorňovat pokud hrozí riziko překročení rychlosti
-    da = Advar hvis der er en risiko for, at hastighedsgrænsen overskrides
-    de = Warnen wenn die Gefahr einer Geschwindigkeitsüberschreitung besteht
-    el = Να ειδοποιούμαι αν υπάρχει κίνδυνος να υπερβώ το όριο ταχύτητας
-    es = Alertar si existe el riesgo de exceder el límite de velocidad
-    es-MX = Avisar si existe el riesgo de exceder el límite de velocidad
+    comment = "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit
+    tags = android,ios
+    en = Warn if exceeding the speed limit
+    af = Waarsku teen spoed
+    ar = التحذير من السرعة
+    az = Sürətlə bağlı xəbərdarlıq edin
+    be = Папярэджваць калі хуткасць перавышана
+    bg = Да ви предупреди за превишена скорост
+    ca = Avisa’m en excedir velocitat límit
+    cs = Varování před překročením rychlosti
+    da = At advare dig om hastighedsoverskridelser
+    de = Um dich vor Geschwindigkeitsüberschreitungen zu warnen
```

Comment:
suggestion by @misalf-git https://github.com/organicmaps/organicmaps/pull/7585#issuecomment-1996138574 :

```suggestion
    de = Warnen vor Geschwindigkeitsüberschreitung
```

or

```suggestion
    de = Tempolimit-Warnung
```

Can you choose/explain @misalf-git?

Reply:
@Misalf-git: First suggestion does not seem to fit in 1 line:

![ios_speed_cam_prefs_de1](https://github.com/organicmaps/organicmaps/assets/25888296/3c366d21-bf49-41af-98c1-9f104f779042)

Please confirm whether we can go with the second option `de = Tempolimit-Warnung`

Reply:
Sorry, I didn't get a ping. Just landed here by pure luck.

Now I seem to better get the context...
Maybe better:
"Warnen, wenn zu schnell"

Reply:
Not sure about comma-correctness as it's not a full sentence anyway.
Maybe omit the comma for better looks.

Reply:
@Misalf-git: Here you go...

![imagen](https://github.com/organicmaps/organicmaps/assets/25888296/f79f5042-b2ae-4155-9121-b87921655127)